src/main/java/frc/robot/commands/ElevatorCalibrationAbsoluteEncoder.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.commands;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.constants.ElevatorConstants;
+import frc.robot.subsystems.Elevator;
+
+public class ElevatorCalibrationAbsoluteEncoder extends CommandBase {
+  Elevator m_elevator; 
+
+  public ElevatorCalibrationAbsoluteEncoder(Elevator elevator) {
+    m_elevator = elevator; 
+    addRequirements(m_elevator);
+  }
+
+  @Override
+  public void initialize() {
+    m_elevator.set(ElevatorConstants.kElevatorMotorEncoderZeroingPower); 
+  }
+
+  @Override
+  public void end(boolean interrupted) {
+    m_elevator.stopMotor();
+    m_elevator.calibrationZeroAbsEncoder();;
+  }
+  
+  // Returns true when the command should end.
+  @Override
+  public boolean isFinished() {
+    if(m_elevator.returnBottomLimSwitchCondition() == true){
+      return true; 
```

Comment:
for these kinds of if statements, you can just do `return m_elevator.returnBottomLimSwitchCondition()` because it will be the same as `return true` or `return false`

Reply:
same for all of the other commands

Reply:
Change Incorporated